tuplex/core/include/logical/TakeOperator.h

```diff
@@ -17,15 +17,16 @@
 namespace tuplex {
     class TakeOperator : public LogicalOperator {
     private:
-        int64_t _limit;
+        int64_t _limitTop;
+        int64_t _limitBottom;
     public:
         LogicalOperator *clone() override;
 
     public:
-        TakeOperator(LogicalOperator *parent, const int64_t numElements);
+        TakeOperator(LogicalOperator *parent, const int64_t numTop, const int64_t numBottom);
```

Comment:
rename to topLimit and bottomLimit, set bottomLimit per default to 0 (indicating no bottom limit), add doxygen string to explain that behavior.